Artifacts/MsSql/Data/Security/1030-DataStandard3.3a-ResourceClaimMetadata.sql

```diff
@@ -6,10 +6,15 @@
     DECLARE @applicationId INT;
     DECLARE @systemDescriptorsResourceClaimId INT;
     DECLARE @relationshipBasedDataResourceClaimId INT;
+    DECLARE @authorizationStrategyId INT;
+    DECLARE @assessmentMetadataResourceClaimId INT;
 
     SELECT @applicationId = (SELECT applicationid FROM  dbo.Applications  WHERE  ApplicationName  = 'Ed-Fi ODS API');
     SELECT @systemDescriptorsResourceClaimId = (SELECT ResourceClaimId FROM dbo.ResourceClaims WHERE ResourceName = 'systemDescriptors' AND Application_ApplicationId = @applicationId);
     SELECT @relationshipBasedDataResourceClaimId = (SELECT ResourceClaimId FROM dbo.ResourceClaims WHERE ResourceName = 'relationshipBasedData' AND Application_ApplicationId = @applicationId);
+    SELECT @assessmentMetadataResourceClaimId = (SELECT ResourceClaimId FROM dbo.ResourceClaims WHERE ResourceName = 'assessmentMetadata' AND Application_ApplicationId = @applicationId);
+    SELECT @authorizationStrategyId= (SELECT AuthorizationStrategyId FROM dbo.AuthorizationStrategies WHERE DisplayName='No Further Authorization Required');
```

Comment:
Please rename this variable to reflect the specific value it contains -- use `noFurtherAuthRequiredAuthorizationStrategyId`

Reply:
Fixed it . Thanks